src/NUnitFramework/framework/Internal/Builders/DefaultSuiteBuilder.cs

```diff
@@ -114,11 +114,9 @@ public TestSuite BuildFrom(ITypeInfo typeInfo, IPreFilter filter)
             }
             catch (Exception ex)
             {
-                var fixture = new TestFixture(typeInfo);
```

Comment:
This call throws again, defeating the catching above.
Therefore I added a new constructor that create an invalid fixture.

Reply:
Hm. It says I marked this as resolved, but I have no recollection of doing that 🤔